library/alloc/src/collections/btree/testing/crash_test_dummy.rs

```diff
@@ -0,0 +1,67 @@
+#[doc(hidden)]
+#[macro_export]
+macro_rules! crash_test_dummy_def {
```

Comment:
I think instead of a macro we can move CLONES and DROPS to Arc<AtomicUsize> fields inside the CrashTestDummy struct and then just have a regular module.

Reply:
I don't understand. How can such fields be initialized and point to something particular for each test case, without a macro? I could do it by explicitly passing an instance of some manager to every CrashTestDummy instance, just like in ord_chaos.rs, but then a simple reference is fine.

Note that all of these tests, including the ones I pilfered the code template from, are totally single threaded. They just use atomics to avoid scary unsafe blocks accessing globals.

Reply:
A reference would also be fine.